andela/ah-scorpion
# #159393859 API Documentation with swagger

**What does this PR do?**
- Create API Documentation with swagger
Description of tasks completed

**How can this PR be manually tested?**
clone this branch.
install all the dependencies from requirements.txt `pip install -r requirements.txt`
Head to your browser to your local server `http://localhost:8000/`

On your browser you should be able to have this view:
![screen shot 2018-08-16 at 18 58 43](https://user-images.githubusercontent.com/22055953/44220182-8c84c080-a186-11e8-88d1-a60118df7ae6.png)


**What are the relevant pivotal tracker stories?**
[#159393859](https://www.pivotaltracker.com/story/show/159393859)
